lib/importer/paper.rb

```diff
@@ -0,0 +1,86 @@
+module Importer
+  class Paper
+    def self.from(result, urls, source)
```

Comment:
Unused method argument - `urls`. If it's necessary, use `_` or `_urls` as an argument name to indicate that it won't be used.